backend/lib/components/Auth0.js

```diff
@@ -105,10 +106,120 @@ const changePassword = async (token, email) => {
   }
 };
 
+const _getUserDetails = async (token, userId) => {
+  try {
+    const res = await axios.request(
+      `${AUTH_DOMAIN}/api/v2/users/${userId}`,
+      getAuthHeaders(token),
+    );
+    return res.data;
+  } catch (err) {
+    return wrapError(err);
+  }
+};
+
+const _updateUser = async (token, userId, payload) => {
+  try {
+    const res = await axios.request(`${AUTH_DOMAIN}/api/v2/users/${userId}`, {
+      json: payload,
+      ...getAuthHeaders(token),
+    });
+    return res.data;
+  } catch (err) {
+    return wrapError(err);
+  }
+};
+
+const _linkAccounts = async (token, rootUserId, targetUserId) => {
+  const parts = targetUserId.split("|"); // "provider|user_id"
+  const payload = {
+    provider: parts[0],
+    user_id: parts[1],
+  };
+  try {
+    const res = await axios.post(
+      `${AUTH_DOMAIN}/api/v2/users/${rootUserId}/identities`,
+      payload,
+      getAuthHeaders(token),
+    );
+    return res.data;
+  } catch (err) {
+    return wrapError(err);
+  }
+};
+
+const _mergeMetadata = async (token, primaryUserId, secondaryUserId) => {
```

Comment:
I don't think we'll need to worry about merging metadata.

From https://auth0.com/docs/users/user-account-linking:

> The user_metadata and app_metadata of the secondary account are discarded

The secondary account in our flow should have no metadata so there's nothing to worry about.

Reply:
You are right Josh, I don't know if we are going to use metadata later or not, but for this feature, we don't need it, I will remove it (keep it somewhere in case we need it later : )